### DISCUSSION TITLE
Drag Gesture Ending Outside Of Browser Window

### DISCUSSION THREAD

**Discussion by fweth** [pmndrs/use-gesture#510] [General] [answered]
Hello, I implemented a mouse drag gesture using `useMove`, the only problem, when I click the mouse button, drag the mouse out of the browser window and release, it doesn't register in the hook.  Is there a way to solve this with `use-gesture` hooks?

[EDIT] Sorry, I didn't want to post in general, now it seems I can't change it anymore?

**Comment by dbismut** [comment, marked as answer]
Not sure I understand. Is there a way to have a sandbox?

**Reply by fweth** [reply]
I meant something like this: https://codesandbox.io/s/nostalgic-glade-467whf

If you drag out of the window and release, it doesn't register the release.  (But I just realized, if I release and don't move, it doesn't register either, so I need another event listener anyway.)  Sorry if this was a stupid post.

**Reply by dbismut** [reply]
Why aren't you using the useDrag hook which is specifically made for this sort of interaction? What happens here is that you're losing capture, which is expected.

**Reply by fweth** [reply]
Thank you so much!
